Meadowlark-js/tests/e2e/automation-setup/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+
+  db-e2e:
+    container_name: postgres-meadowlark-e2e
+    image: postgres:14.3-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=abcdefgh1!
+    ports:
+      - 5433:5432
```

Comment:
Can the external port be provided with an environment variable, defaulting to 5433? That way it is easy to use a `.env` file to setup a different port number if we're already using 5433.

Reply:
Done